json_schemas/framework-agreement-details.json

```diff
@@ -15,6 +15,57 @@
       "minLength": 1,
       "type": "string"
     },
+    "frameworkRefDate": {
```

Comment:
So to answer your question `dateForframeworkAgreementReferenceNumber` would be more descriptive, but I'm not sure it really helps :/